google-cloud-language/lib/google/cloud/language/v1/language_service/client.rb

```diff
@@ -0,0 +1,1005 @@
+            #   @param options [Gapic::CallOptions, Hash]
+            #     Overrides the default settings for this call, e.g, timeout, retries, etc. Optional.
+            #
+            # @overload analyze_sentiment(document: nil, encoding_type: nil)
+            #   @param document [Google::Cloud::Language::V1::Document | Hash]
+            #     Input document.
+            #   @param encoding_type [ENUM(EncodingType)]
+            #     The encoding type used by the API to calculate sentence offsets.
+            #
+            #
+            # @yield [response, operation] Access the result along with the RPC operation
+            # @yieldparam response [Google::Cloud::Language::V1::AnalyzeSentimentResponse]
+            # @yieldparam operation [GRPC::ActiveCall::Operation]
+            #
+            # @return [Google::Cloud::Language::V1::AnalyzeSentimentResponse]
+            #
+            # @raise [Google::Cloud::Error] if the RPC is aborted.
+            #
+            # @example Analyzing Sentiment in text file stored in Cloud Storage
+            #   require "google/cloud/language/v1/language_service"
+            #
+            #   client = Google::Cloud::Language::V1::LanguageService::Client.new
+            #
+            #   gcs_content_uri = "gs://cloud-samples-data/language/sentiment-positive.txt"
+            #
+            #   document = {
+            #     # Google Cloud Storage URI where the file content is located.
+            #     # e.g. gs://[Your Bucket]/[Path to File]
+            #     gcs_content_uri: gcs_content_uri,
+            #     # Available types: PLAIN_TEXT, HTML
+            #     type: :PLAIN_TEXT,
+            #     # Optional. If not specified, the language is automatically detected.
+            #     # For list of supported languages:
+            #     #   https://cloud.google.com/natural-language/docs/languages
+            #     language: "en"
+            #   }
+            #   # Available values: NONE, UTF8, UTF16, UTF32
+            #   encoding_type = :UTF8
+            #
+            #   response = client.analyze_sentiment document: document, encoding_type: encoding_type
+            #
+            #   # Get overall sentiment of the input document
+            #   puts "Document sentiment score: #{response.document_sentiment.score}"
+            #   puts "Document sentiment magnitude: #{response.document_sentiment.magnitude}"
+            #
+            #   # Get sentiment for all sentences in the document
+            #   response.sentences.each do |sentence|
+            #     puts "Sentence text: #{sentence.text.content}"
+            #     puts "Sentence sentiment score: #{sentence.sentiment.score}"
+            #     puts "Sentence sentiment magnitude: #{sentence.sentiment.magnitude}"
+            #   end
+            #
+            #   # Get the language of the text, which will be the same as
+            #   the language specified in the request or, if not specified,
```

Comment:
These additional comment lines are not commented out. We should probably split on `\n` to make sure each line gets a comment character in front of it.